### ISSUE TITLE
2ch switch module TS0002 / _TZ3000_zmy4lslw, missing power_on_behavior and switch_type

### ISSUE THREAD

**Issue by ra-kal** [Koenkk/zigbee-herdsman-converters#5040]
This is continuation of  closed #5032. I've tried the dev branch - the HA edge version. But the changes seems not to work correctly.

I removed the device from network and joined it again. But the changes seems not to work correctly.

- there is no change on the exposes page, e.g. no possibility to configure power_on_behavior and switch_type
- there is no possibility to configure these options in HA either.
- state has additional "power_on_behavior_l1", missing is "switch_type"
```
{
"last_seen": 1669675010137,
"linkquality": 132,
"power_on_behavior_l1": "off",
"state_l1": "ON",
"state_l2": "OFF",
"elapsed": 55
}
```
I used dev console to change the power_on_behavior_l1 to off (1/onoff/moesStartUpOnOff = 0) and it seems to stay off at power on. Value 1 caused to stay at On and Value 2 to switch on or off based on the switch position (and not the last state).

Moreover, depending on the state of the switch before power off, I have to change the position of the switch twice for the first time after power on, in order the change the output state. Actually I'm not sure if I saw the same behavior on the other modules.

On my 1 channel switch the state is as follows:
```
{
"last_seen": 1669675022590,
"linkquality": 164,
"power_on_behavior": "off",
"state": "OFF",
"switch_type": "toggle"
}
```

On my 3 channel switch the state is as follows:
```
{
"last_seen": 1669675367309,
"linkquality": 220,
"power_on_behavior": "off",
"power_on_behavior_l1": "off",
"state_l1": "OFF",
"state_l2": "OFF",
"state_l3": "OFF",
"switch_type": "toggle"
}
```

**Comment by ra-kal** [comment]
OK, so I modified tuya.js and added manufacturer to the fingerprint and voala, now it works as expected both in Z2M web and HA. Both power_on_behavior and switch type can be configured.

```
    {
        fingerprint: [{modelID: 'TS0002', manufacturerName: '_TZ3000_01gpyda5'}, {modelID: 'TS0002', manufacturerName: '_TZ3000_bvrlqyj7'},
            {modelID: 'TS0002', manufacturerName: '_TZ3000_7ed9cqgi'}, **{modelID: 'TS0002', manufacturerName: '_TZ3000_zmy4lslw'}**],
        model: 'TS0002_switch_module',
        vendor: 'TuYa',
        description: '2 gang switch module',
        whiteLabel: [{vendor: 'OXT', model: 'SWTZ22'}],
        extend: tuya.extend.switch({switchType: true, endpoints: ['l1', 'l2']}),
        endpoint: (device) => {
            return {'l1': 1, 'l2': 2};
        },
        meta: {multiEndpoint: true},
        configure: async (device, coordinatorEndpoint, logger) => {
            await tuya.configureMagicPacket(device, coordinatorEndpoint, logger);
            await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
            await reporting.bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
        },
    },

```

PS: The behaviour at power on is still a mystery. It must be the module itself. If the switch was on before power off, then after power on the first button change is not registered. I have to try it with the 1ch and 3ch modules again, but  can't remember that I would notice such behavior.

**Comment by Koenkk** [comment]
This should already be available on the latest dev. How does the definition (in `tuya.js`) show up with the latest dev?

**Comment by ra-kal** [comment]
It founds nothing if I search for "_TZ3000_zmy4lslw" in tuya.js. In current version it looks like this:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/a8d3bf21b32560f4846e4ac737367898955a3a89/devices/tuya.js#L1811

**Comment by Koenkk** [comment]
Added it now!

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Comment by pipiche38** [comment]
@Koenkk hello guys, I'm looking for help/advise.
I have a TS0002 / _TZ3000_zmy4lslw for which when I switchOn one relay , the two relays switchOn, then if I switchOff 1 relay, the two switchOff.

Is that something talking to you ?

**Comment by Koenkk** [comment]
@pipiche38 if using z2m, this should already be fixed in z2m 1.29.1 (it required the magic tuya packet to be send to fix this).

**Comment by pipiche38** [comment]
> @pipiche38 if using z2m, this should already be fixed in z2m 1.29.1 (it required the magic tuya packet to be send to fix this).

Thanks for responding. I found it
